src/html-file/html-file.component.js

```diff
@@ -93,7 +94,7 @@ function useCode() {
       })
     </script>
     <!-- See https://github.com/joeldenning/import-map-overrides#user-interface  -->
-    <import-map-overrides-full></import-map-overrides-full>
+    <import-map-overrides-full show-when-local-storage="overrides-ui"></import-map-overrides-full>
```

Comment:
See [release notes](https://github.com/joeldenning/import-map-overrides/releases/tag/v1.7.0) for what `show-when-local-storage` does.